src/Raven.Client/Documents/Session/DocumentSession.cs

```diff
@@ -148,6 +148,19 @@ public void Refresh<T>(T entity)
             RefreshInternal(entity, command, documentInfo);
         }
 
+        /// <summary>
+        /// Refreshes the specified entities from Raven server.
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="entities">The entities.</param>
+        public void Refresh<T>(T[] entities)
```

Comment:
That is a *bad* idea.
The problem is that this will issue `N` requests to the server, while from the API, the user can reasonably expect that this would be done in 1 request to the server

Reply:
changed it. 
now a single request sent to the server. 
`GET /databases/TestRefreshOverloadAsync_1/docs?&id=TestDoc0&id=TestDoc1&id=TestDoc2 HTTP/1.1`